README.md

```diff
@@ -1,7 +1,64 @@
+<img src="./relaynet-logo.png" align="right"/>
+
 # Relaynet Gateway for Android
 
-This repository contains the source code for the Relaynet Gateway for Android, which is all Relaynet users need on their Android devices. To learn more about using Relaynet, visit [relaynet.network](https://relaynet.network/users).
+The [Relaynet Gateway for Android](https://play.google.com/store/apps/details?id=tech.relaycorp.gateway) is a _[private gateway](https://specs.relaynet.network/RS-000#concepts)_ for Android 5+ devices. This repository contains the source code for the app, which is also a reference implementation of a private gateway in the Relaynet protocol suite.
+
+This document is aimed at advanced users and (prospective) contributors. We aim to make the app as simple and intuitive as possible, and we're therefore not planning on publishing end-user documentation at this point. To learn more about _using_ Relaynet, visit [relaynet.network/users](https://relaynet.network/users).
+
+## Relaynet bindings
+
+This private gateway implements [Relaynet bindings](https://specs.relaynet.network/RS-000#message-transport-bindings) as follows:
+
+- Local endpoints communicate with the private gateway via a [PoWeb](https://specs.relaynet.network/RS-016) server on `127.0.0.1:13276`. This server is implemented with the PoWeb binding and [ktor](https://ktor.io).
+- When the Internet is available and the public gateway is reachable, this private gateway will communicate with its public counterpart using [Relaycorp's PoWeb client](https://docs.relaycorp.tech/relaynet-poweb-jvm/).
+- When communicating with couriers over WiFi, this private gateway uses the [CogRPC binding](https://specs.relaynet.network/RS-008) through [Relaycorp's CogRPC client](https://docs.relaycorp.tech/relaynet-cogrpc-jvm/).
+
+The local communication with endpoints does not use TLS, but all other connections are external and therefore require TLS.
+
+By default, instances of this gateway are paired to [Relaycorp's Frankfurt gateway](https://github.com/relaycorp/cloud-gateway/tree/main/environments/frankfurt).
+
+## Security and privacy considerations
+
+The items below summarize the security and privacy considerations specific to this app. For a more general overview of the security considerations in Relaynet, please refer to [RS-019](https://specs.relaynet.network/RS-019).
+
+### No encryption at rest on Android 5
```

Comment:
I was not familiar with the expression "at rest" in this context, but it looks to be a known term in computer science.